static/src/javascripts/projects/common/views/commercial/creatives/fabric.html

```diff
@@ -0,0 +1,17 @@
+<div
+    class="fabric"
+    style="
+        background-image: url(<%=backgroundImage%>);
+        background-color: <%=backgroundColour%>;
+        background-repeat: <%=backgroundRepeat%>;
+        background-position: <%=backgroundPosition%>;
+        background-attachment: <%=(backgroundScroll === 'reveal' ? 'fixed' : 'scroll')%>;
```

Comment:
So, we've seen this issue before?

Possibly I will end up moving this logic into some kind of proper viewmodel anyway.